checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,705 @@
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `event_type` to `checkpoint_event_type`
+          enabled: true
+          meta:
+            tags: []
+          sources:
+            - event_type
+          sourceType: attribute
+          target: checkpoint_event_type
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Remapper for Username Attribute
+      enabled: true
+      filter:
+        query: "@product:DLP "
+      processors:
+        - type: attribute-remapper
+          name: Map `user` to `usr.name`
+          enabled: true
+          meta:
+            tags: []
+          sources:
+            - user
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: String Building for Source and Destination IP Port
+      enabled: true
+      filter:
+        query: '@product:("Firewall" OR "IPS" OR "Anti-Spam and Email Security" OR "URL
+          Filtering") '
+      processors:
+        - type: string-builder-processor
+          name: "Define `src_ip_port` equal to %{network.client.ip}:
+            %{network.client.port}"
+          enabled: true
+          meta:
+            tags: []
+          template: "%{network.client.ip}:%{network.client.port}"
+          target: src_ip_port
+          replaceMissing: false
+        - type: string-builder-processor
+          name: "Define `dst_ip_port` equal to %{network.destination.ip}:
+            %{network.destination.port}"
+          enabled: true
+          meta:
+            tags: []
+          template: "%{network.destination.ip}:%{network.destination.port}"
+          target: dst_ip_port
+          replaceMissing: false
+    - type: pipeline
+      name: Lookup for `app_risk` field
+      enabled: true
+      filter:
+        query: '@product:("Application Control" OR "URL Filtering") '
+      processors:
+        - name: Lookup for `app_risk` to `app_risk_str` field
+          enabled: true
+          meta:
+            tags: []
+          source: app_risk
+          target: app_risk_str
+          lookupTable: |-
+            0, Unknown
+            1, Very low
+            2, Low
+            3, Medium
+            4, High
+            5, Critical
+            Unknown, Unknown
+            Very low, Very low
+            Low, Low
+            Medium, Medium
+            High, High
+            Critical, Critical
+          type: lookup-processor
+    - type: pipeline
+      name: Remapper for Status Attribute
+      enabled: true
+      filter:
+        query: '@product:"HTTPS Inspection" '
+      processors:
+        - type: attribute-remapper
+          name: Map `status` to `checkpoint_firewall_status`
```

Comment:
Why not keeping the status as the standard `status` attribute ? 

If the status doesn't match datadog statuses, you can use a category processor to transform the values.

Reply:
We have mapped status to checkpoint.quantum.firewall.status because the value of status were succeeded, failed, etc which were not matching with the reserved attribute status. Also, the category processor were not clear to define based on the values.

Reply:
Status is a very useful in datadog's UI. Succeeded and failed seem like the kind of values that could very well be mapped to the standard attribute.